modules/sage-aws-k8s-node-autoscaler/templates/values.yaml

```diff
@@ -100,6 +110,8 @@ securityContext:
     drop:
     - ALL
 
+command: []
```

Comment:
What does this mean here? Does it mean to use the default entry point?

Reply:
Ah got it. I saw it on the UI too: 
<img width="1388" height="761" alt="Screenshot 2025-09-04 at 3 12 05 PM" src="https://github.com/user-attachments/assets/ab473e6b-6383-4826-bf77-1b897c324c34" />

Reply:
Do we usually just copy the template that they provide? I can find template here: https://artifacthub.io/packages/helm/spot/ocean-kubernetes-controller